mkt/reviewers/utils.py

```diff
@@ -682,6 +684,28 @@ def get_pending_queue(self):
             addon__disabled_by_user=False,
             addon__status=mkt.STATUS_PENDING)
             .exclude(addon__id__in=self.excluded_ids)
+            .exclude(addon__tags__tag_text='homescreen')
```

Comment:
Updated homescreens now go in homescreen queue; re-reviews/escalations for homescreens go in app re-review/escalation queues.